### ISSUE TITLE
Cannot login after upgrading 

### ISSUE THREAD

**Issue by TJC129** [sameersbn/docker-gitlab#330]
Hi sameersbn,
Thank you for your awesome work!! This repo saved a newbie like me.
I'm currently having a little problem after upgrading, could you please help? 
Originally, I was using a one liner to run gitlab like 
```
sudo docker run --name='gitlab' -it --rm \
-e 'GITLAB_PORT=10080' -e 'GITLAB_SSH_PORT=10022' \
-p 10022:22 -p 10080:80 \
-v /var/run/docker.sock:/run/docker.sock \
-v $(which docker):/bin/docker \
-v /placetoStoreData:/home/git/data \
-e 'SMTP_USER=TJC@something.com.tw' -e 'SMTP_PASS=secretPW' \
-e 'GITLAB_HOST=someddns.com' \
sameersbn/gitlab
```
after upgrading redis, postgradesql, gitlab,
I use the following cmd instead
```
docker run --name=postgresql-gitlab -d \
  --env='DB_NAME=gitlabhq_production' \
  --env='DB_USER=gitlab' --env='DB_PASS=password' \
  --volume=/srv/docker/gitlab/postgresql:/var/lib/postgresql \
  sameersbn/postgresql:9.4
  
  
docker run --name=redis-gitlab -d \
  --volume=/srv/docker/gitlab/redis:/var/lib/redis \
  sameersbn/redis:latest
  
  
docker run --name='gitlab' -d \
  --link=postgresql-gitlab:postgresql --link=redis-gitlab:redisio \
  --publish=10022:22 --publish=10080:80 \
  --env='GITLAB_PORT=10080' --env='GITLAB_SSH_PORT=10022' \
  --volume=/placetoStoreData:/home/git/data \
-e 'SMTP_USER=TJC@something.com.tw' -e 'SMTP_PASS=secretPW' \
-e 'GITLAB_HOST=someddns.com' \
sameersbn/gitlab:7.11.2
```
now I cannot login with my previous pw/ email... could you please help to point out the problem?

Thank you in advance!

TJC

**Comment by likol** [comment]
I think if you want to upgrade gitlab, just need remove old gitlab container,you don't need remove all .
If you can't login with existed accounts, maybe lost database data.

be sure when you create new postgreSQL container, the data volume location is same as old container. :smiley:

**Comment by TJC129** [comment]
Hi Likol,
Thank you very much for your reply!
I can still find the old git repositories data at ``` --volume=/placetoStoreData: ```
So, I guess I messed up something with the other two containers (removed and pulled all three containers...) like you said.


maybe I should try to find some way to get these data back to work

**Comment by sameersbn** [comment]
@TJC129 Unless you had taken backups or enabled automatic backups, you have lost your postgresql database. The command that you used was for demonstration purposes only. When used in production you were required to manually create postgresql and redis containers.

If you have a backup, then you can start the container and restore the backup and then upgrade.

**Comment by TJC129** [comment]
Hi @sameersbn, thank you for your reply.
I think have already lost the database(my fault :P), should think before leap next time.
sorry for interrupt again, now I'm trying to use rake:import function 
```
docker exec -it gitlab sudo -u git -H bundle exec rake gitlab:import:repos RAILS_ENV=production
```
with a successful response like 

``` 
* Skipping wiki repo
Processing backup/plcdriver.git
 * Created plcdriver (backup/plcdriver.git)
Done!
```
but they turns out to be empty repos
![empty](https://cloud.githubusercontent.com/assets/3121302/7875474/782e5c5c-05f0-11e5-9a70-6da727d57b21.png)
Any other options should we check before doing import?

**Comment by sameersbn** [comment]
> I think have already lost the database(my fault :P), should think before leap next time.

This is the reason why I have removed the docker socket mounting in quickstart. Far too many people seem to have considered the quick start a be-all and end-all.

> but they turns out to be empty repos

Are these repos empty when you clone them? it might just be that they are very small for gitlab to count them in MB's

**Comment by TJC129** [comment]
Hi @sameersbn ,
I'm sorry for this late and Thank you for your help again! 
I just found that the repo cloned from the "empty repo" is not empty:P.
The size's about 20 MB's, and after I made some another push, the size label on Web GUI became normal again. So I guess the import is quite success, unless the web GUI didn't display the correct size at first due to my missing postgresql/redis.

Thank you again for the hint, this awesome repo, and the modified quick start script! It will save a lot of newbie like me XD.
